src/components/MetadataForm/MetadataFormStyles.ts

```diff
@@ -1,7 +1,10 @@
 import styled from 'styled-components';
 
-export const DummyFormStyle = styled.form`
+export const MetadataFormStyle = styled.form`
 	display: flex;
 	flex-direction: column;
-	margin: 10px 50px;
+
+	h2 {
+		margin-top: 0;
+	}
```

Comment:
Waarom dit helemaal in een aparte module stoppen? kan ook in MetadataForm.tsx toch?

Waarom via tags (h2) stylen en niet via component composition (dus via `<Header level={2} />` oid)?

Deze manier gaat een beetje tegen het concept van styled components in.

Reply:
Fair point :+1: 
Fixed.